Simulator/Edges/Neuro/AllDSSynapses.h

```diff
@@ -49,7 +49,7 @@
 
 #include "AllSpikingSynapses.h"
 // cereal
-#include <cereal/archives/xml.hpp>   // this is a cereal hack
+// #include <cereal/archives/xml.hpp>   // this is a cereal hack
```

Comment:
I am so sorry about this, but this cereal hack should not be commented. I forgot to uncomment this statement when pushing my change.

Reply:
OK, adding this to the review.